src/deploy.py

```diff
@@ -94,15 +94,18 @@ def configure_montagu(service, is_first_time):
         data_import.do(service)
     orderly.configure_orderly(service, not data_exists)
 
-    passwords = database.setup(service)
+    annex_settings = database.setup_annex(service)
+    passwords = database.setup(service, annex_settings)
+    annex_root_user = annex_settings['users']['root']
 
     # Push secrets into containers
     cert_paths = get_ssl_certificate(service.settings["certificate"])
     token_keypair_paths = get_token_keypair()
 
     send_emails = service.settings["password_group"] == 'production'
+    add_annex = service.settings["db_annex_type"] != 'readonly'
     configure_api(service, passwords['api'], token_keypair_paths,
-                  service.settings["hostname"], send_emails)
+                  service.settings["hostname"], send_emails, add_annex, annex_root_user.name, annex_root_user.password)
```

Comment:
Why not just pass the user object through as a single argument, instead of splitting it up into separate username and password args?